metadata/schemas/sensor.json

```diff
@@ -51,32 +51,22 @@
                 "type": {
                     "type": "string",
                     "enum": [
-                        "temperature", "displacement", "radioactivity", "humidity"
+                        "temperature",
+                        "displacement",
+                        "radioactivity",
+                        "humidity"
                     ]
                 },
                 "unit": {
                     "description": "SI unit.",
                     "type": "string"
-                }
-            }
-        },
-        "independent_variable": {
-            "description": "The chosen variable to make the measurements as a function of.",
-            "type": "object",
-            "required": [
-                "type",
-                "unit"
-            ],
-            "properties": {
-                "type": {
-                    "type": "string",
-                    "enum": [
-                        "time"
-                    ]
                 },
-                "unit": {
-                    "description": "SI unit.",
-                    "type": "string"
+                "values": {
+                    "description": "List of measured values",
+                    "type": "array",
+                    "items": {
+                        "type": "number"
```

Comment:
is number a real/double/float or int? Usually, this is distinguished.

Reply:
number is used in json schema for int/float, seems it can't [differentiate](https://json-schema.org/understanding-json-schema/reference/type.html#id4)